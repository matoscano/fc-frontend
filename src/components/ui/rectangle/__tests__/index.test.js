import React from "react";
import { render } from "@testing-library/react";
import Rectangle from "../index";

it("renders a Rectangle", () => {
  const { container } = render(<Rectangle>Rectangle content</Rectangle>);
  expect(container).toMatchSnapshot();
});
