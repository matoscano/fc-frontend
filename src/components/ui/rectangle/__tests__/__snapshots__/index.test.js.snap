// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a Rectangle 1`] = `
<div>
  <div
    class="sc-AxjAm juRXTL"
  >
    Rectangle content
  </div>
</div>
`;
